syncer/ddl_test.go

```diff
@@ -35,7 +40,32 @@ import (
 	"go.uber.org/zap"
 )
 
-func (s *testSyncerSuite) TestAnsiQuotes(c *C) {
+var _ = Suite(&testDDLSuite{})
+
+type testDDLSuite struct{}
+
+func (s *testDDLSuite) newSubTaskCfg(dbCfg config.DBConfig) *config.SubTaskConfig {
+	return &config.SubTaskConfig{
+		From:             dbCfg,
+		To:               dbCfg,
+		ServerID:         101,
+		MetaSchema:       "test",
+		Name:             "syncer_ut",
+		Mode:             config.ModeIncrement,
+		Flavor:           "mysql",
+		ShadowTableRules: []string{config.DefaultShadowTableRules},
+		TrashTableRules:  []string{config.DefaultTrashTableRules},
+	}
+}
+
+func (s *testDDLSuite) mockParser(db *sql.DB, mock sqlmock.Sqlmock) (*parser.Parser, error) {
```

Comment:
if `GetParser` is covered by other tests, we can directly return a Parser :thinking: